lib/std/c.zig

```diff
@@ -5598,6 +5598,7 @@ pub const MSG = switch (native_os) {
     .linux => linux.MSG,
     .emscripten => emscripten.MSG,
     .windows => ws2_32.MSG,
+    .macos, .ios, .tvos, .watchos, .visionos => darwin.MSG,
```

Comment:
Needs `driverkit` too.

```suggestion
    .driverkit, .macos, .ios, .tvos, .watchos, .visionos => darwin.MSG,
```